docs/02-QuickStart/04-DeveloperSetup.md

```diff
@@ -31,26 +31,31 @@ description: >
     ```sh
 	go get
     ```
+   
+4. Replace the devlake module with the current project path
```

Comment:
Do we need this? I have setup devlake multiple times, it worked fine without this operation.

Reply:
Mainly for a build of the user, according to the official website after installation, can work normally, but IDE such as my Goland will show the import dependency package error

Reply:
I see, did you try to restart your IDE? This looks like a workaround for IDE to me, not a legitimate problem of DevLake or Golang itself.
Will this operation modify any file like `go.mod` ?
If it happens only for some IDEs, we should move it to `Trouble-shooting` part, and describe the consequence of this workaround.